### PR TITLE
Update summary cards each 30 seconds

### PR DESCRIPTION
**Pull request by ramirotw**
# Summary

Sets an interval to fetch the summary data each ~~30~~ 10 seconds

### CODE REVIEW

File: src/apps/explorer/components/SummaryCardsWidget/useGetSummaryData.ts [hunk 1]
```diff
@@ -83,15 +83,24 @@ export function useGetSummaryData(): TotalSummaryResponse | undefined {
...
+    const summary = buildSummary(data)
+    setSummary({ ...summary, isLoading: false })
   }, [network])
 
+  useEffect(() => {
+    fetchAndBuildSummary()
+
+    const id = setInterval(() => {
+      fetchAndBuildSummary()
+    }, 1000 * 30) // 30 seconds
```
Comment (matextrem) [inline src/apps/explorer/components/SummaryCardsWidget/useGetSummaryData.ts:99]: Maybe we could move this to be a constant

**Review by matextrem — APPROVED** [review]

**Review by alfetopito — APPROVED** [review]
👍

**Comment by elena-zh** [comment]
Hey @ramirotw , shouldn't CoW Protocol volume card be updated as well?

**Comment by ramirotw** [comment]
> Hey @ramirotw , shouldn't CoW Protocol volume card be updated as well?

Maybe is an overkill, the daily volume is grouped by hour but then the rest is by day, we won't see any updates on the chart unless that timeframe is reached

**Comment by elena-zh** [comment]
@ramirotw , So then maybe we should update a daily volume card once in an hour? 
@alfetopito , WDYT?

**Comment by alfetopito** [comment]
As mentioned yesterday during our sync call, we won't update the graph automatically for now

**Review by elena-zh — APPROVED** [review]

**Review by anxolin — APPROVED** [review]
Great!

Why 30s? I think is not that bad if we query every 10s or so. Or on every block.
With 30s is possible we take almost a minute to update the latest settlement (if we are unlucky)


Also, something unrelated. Don't we plan to add a link to Etherscan (to the token). I think is nice for the meantime while we don't do a "token page"

**Comment by anxolin** [comment]
Question, why there's so many WETH? 
image

**Comment by elena-zh** [comment]
@anxolin , in this PR you see a mocked table. Tokens table is implemented in #98 PR. So

> Also, something unrelated. Don't we plan to add a link to Etherscan (to the token). I think is nice for the meantime while we don't do a "token page"

 it is implemented in #98 

> Question, why there's so many WETH?

 also is changed in #98

**Comment by ramirotw** [comment]
> Why 30s? I think is not that bad if we query every 10s or so. Or on every block.
> With 30s is possible we take almost a minute to update the latest settlement (if we are unlucky)

makes sense. I chose 30 as a starting point and to get some feedback. I'll update it to 10

**Pull request merged**